valmlord/gulpify
# feat: add tasks for optimize images

## Description
This pull request enhances the gulp boilerplate workflow by adding tasks specifically designed for optimizing images. It includes the integration of `gulp-imagemin` and `gulp-newer` to efficiently process image files.

## Changes
- Added `gulp-imagemin` and `gulp-newer` packages to `package.json`.
- Created new Gulp tasks to optimize images using `gulp-imagemin`.
- Configured the `gulp-newer` plugin to ensure that only modified images are processed during builds.

## How to Test
1. Place image files in the appropriate directory  `app/assets/img`.
3. Execute the relevant Gulp task `gulp compressIMG` to optimize the images.
4. Verify that the optimized images are correctly generated in the ` build/assets/images` directory.

## Additional Notes
- This update aims to improve performance by reducing image file sizes while maintaining quality.
